README.md

```diff
@@ -1,67 +1,74 @@
+Iroha can help you manage your accounts, assets, on-chain data storage with efficient smart contracts, while being Byzantine- and crash-fault tolerant and being able to withstand a fault rate of 33%.
 
 ## Features
 
-Iroha is a fully-featured blockchain ledger
+Iroha is a fully-featured blockchain ledger. With Iroha you can:
+
+* Create and manage custom fungible assets, such as currencies, gold, and others
+* Create and manage non-fungible assets
+* Manage user accounts with a domain hierarchy and multi-signature transactions
+* Use efficient portable smart contracts implemented either via WebAssembly or Iroha Special Instructions
+* Use both permissioned and permission-less blockchain deployments
+
+Iroha also offer you:
 
-* Creation and management of custom fungible assets, such as currencies, gold, etc.
-* Non-fungible asset support.
-* User account management, with a domain hierarchy, and multi-signature transactions.
-* Efficient portable smartcontracts implemented either via WebAssembly, or Iroha Special Instructions.
-* Support for both permissioned, and permission-less blockchain deployments.
-* Byzantine fault-tolerance with up to 34% fault rate.
-* Efficient in-memory operation.
-* Extensive telemetry support out of the box.
-* Modular structure. Don't need it? Compile with `--no-default-features` and add the features you need.
-* Event-driven architecture with strongly-typed events.
+* Byzantine fault-tolerance with up to 34% fault rate  
+* Efficient in-memory operations
+* Extensive telemetry support out of the box
+* Modular structure that can be easily customized with `--no-default-features`
+* Event-driven architecture with strongly-typed events
 
 # System Requirements
 
-| Use-case          | CPU               | RAM   | Storage[^1] |
+RAM and storage requirements depend on your use case: whether you need to build or deploy a network, how big it is, and so on. This table summarises the requirements:
+
+| Use case          | CPU               | RAM   | Storage[^1] |
 |-------------------|-------------------|-------|-------------|
 | Build (minimum)   | Dual-core CPU     | 4GB   | 20GB        |
 | Build (recommend) | AMD Ryzen™ 5 1600 | 16GB  | 40GB        |
 | Deploy (small)    | Dual-core CPU     | 8GB+  | 20GB+       |
 | Deploy (large)    | AMD Epyc™ 64-core | 128GB | 128GB+      |
 
-[^1]: Note, all operations are done in RAM, so it can theoretically work without persistent storage. However, since synchronising blocks can take a long time, we recommend adding a hard drive.
+[^1]: Note that all operations are done in RAM, so in theory Iroha can work without persistent storage. However, since synchronising blocks may take a long time, we recommend adding a hard drive.
```

Comment:
```suggestion
[^1]: Note that all operations are done in RAM, so in theory Iroha can work without persistent storage. However, since synchronising blocks over the network after a power failure may take a long time, we recommend adding a hard drive.
```